apis/python/tests/test_coordinates.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import pytest
+from somacore import (
+    AffineTransform,
+    Axis,
+    CoordinateSpace,
+    CoordinateTransform,
+    IdentityTransform,
+)
+
+import tiledbsoma as soma
+
+
+def check_transform_is_equal(
+    actual: CoordinateTransform, desired: CoordinateTransform
+) -> None:
+    assert actual.input_axes == desired.input_axes
+    assert actual.output_axes == desired.output_axes
+    if isinstance(desired, IdentityTransform):
```

Comment:
nit for the future: `expected` is a more standard wording than `desired` (although the latter is more fresh & creative :) )

Reply:
I can't take the credit - I store the name from [numpy](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html)

Reply:
OK TIL, thank you! :)